api/migrations/00001_create_sources_table.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS sources(
+id SERIAL
+source_id  INT PRIMARY KEY,
+name VARCHAR(100) NOT NULL,
+auth_id INT, 
+org_id INT,
```

Comment:
Should we have index on `org_id`?

Reply:
Yes :)
 I have a few bugs I am fixing and then I'll push the updated version